packages/gafl-webapp-service/src/locales/en.json

```diff
@@ -103,7 +103,7 @@
   "age_concession": "Age concession",
   "age_junior_concession": "Junior (13 to 16)",
   "age_junior": "Junior",
-  "age_senior_concession": "Over 65",
+  "age_senior_concession": "Senior",
```

Comment:
do we need to do something with how it displays? previously it was for "over 65, type" but now we need it to be "type (senior)"

Reply:
Yeah, you're right, I'm rushing this, hadn't noticed that the sentence order had switched too